packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue

```diff
@@ -57,31 +53,28 @@ export default {
   },
   mounted() {
     this.$on("toggle", this.toggleHandler);
+    this.setAccordionItems();
     this.setAsOpen();
     this.$emit("click:open-header");
   },
   updated() {
+    this.setAccordionItems();
     this.setAsOpen();
     this.$emit("click:open-header");
   },
   methods: {
+    setAccordionItems() {
+      if (this.$children && this.$children.length) {
+        this.items.push(...this.$children);
+      }
+    },
     setAsOpen() {
       if (this.$children && this.$children.length) {
-        // TODO remove in 1.0.0 ->
```

Comment:
Just a question that isn't related to memory leak - should we remove it now? There will be `0.11.4` version and here we declared that it will be removed in `1.0.0` version? :slightly_smiling_face:

Reply:
we don't really use it already, and this todo was added almost 2 years ago, I think its safe to do it :)